tmt/export/polarion.py

```diff
@@ -414,9 +521,466 @@ def export_to_polarion(test: tmt.base.Test) -> None:
+
+    polarion_feature = None
+    if not duplicate:
+        polarion_feature = get_polarion_feature(story.node, project_id, polarion_feature_id)
+    
+    # Prepare summary
+    summary = tmt.export.nitrate.prepare_extra_summary(story, append_summary)
+
+    if not polarion_feature:
+        if create:
+            if not project_id:
+                raise ConvertError(
+                    "Please provide project_id so tmt knows which "
+                    "Polarion project to use for this feature."
+                )
+            if not dry_mode:
+                polarion_feature = create_polarion_feature(
+                    summary, project_id=project_id, story_text=story.story or ""
+                )
+            else:
+                echo(style(f"Feature '{summary}' created.", fg='blue'))
+            story._metadata['extra-summary'] = summary
+        else:
+            raise ConvertError(
+                f"Polarion feature id not found for '{story}'. "
+                f"(You can use --create option to enforce creating features.)"
+            )
+
+    # Title
+    if not dry_mode:
+        assert polarion_feature  # Narrow type
+        if story.title is not None and polarion_feature.title != story.title:
+            polarion_feature.title = story.title
+        elif story.summary is not None and polarion_feature.title != story.summary:
+            polarion_feature.title = story.summary
+    title = story.title or story.summary
+    if title:
+        logger.debug(f"title: {title}")
+
+    # Add id to story
+    uuid = add_uuid_if_not_defined(story.node, dry_mode, story._logger)
+    if not uuid:
+        uuid = story.node.get(ID_KEY)
+    logger.debug(f"Appended ID: {uuid}")
+    
+    # Store Polarion work item ID in extra-polarion for future lookups
+    if not dry_mode and polarion_feature:
+        assert polarion_feature  # Narrow type
+        polarion_work_item_id = str(polarion_feature.work_item_id)
+        with story.node as data:
+            data['extra-polarion'] = polarion_work_item_id
+        logger.info(f"Stored Polarion work item ID: {polarion_work_item_id}")
+
+    # Description (story text + description)
+    # Convert Markdown to HTML for rich text formatting
+    
+    # Add header indicating TMT autogeneration with link to metadata
+    description = ""
+    web_link = story.web_link()
+    if web_link:
+        # Use the actual origin repository URL without any hardcoded modifications
+        description = (
+            f'<p><em>ℹ️ This feature is autogenerated by TMT from metadata: '
+            f'<a href="{web_link}">{story.name}</a></em></p>'
+            f'<hr/>'
+        )
+    
+    if story.story:
+        description += markdown_to_html(story.story)
+    if story.description:
+        if description and not web_link:  # Only add spacing if no header was added
+            description += '<br/><br/>'
+        elif description:  # Header was added
+            description += '<br/>'
```

Comment:
shouldn't one line be enough for both of these cases? even where there would be a web_link, then that part is before the story, so IMO doesn't matter here